### PR TITLE
updated sight.vue

### PR DESCRIPTION
**Pull request by shaista1519**

### CODE REVIEW

File: pages/sight.vue [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
   <div>
     <div>
-      <spacer gap="3" />
       <div class="d-flex flex-column justify-space-between align-center">
         <v-img
           contain
-          src="https://firebasestorage.googleapis.com/v0/b/ieee-amu.appspot.com/o/amu-logo.png?alt=media&token=fe5c2453-4526-4934-8679-0ca61b0883f8"
-          width="250px"
-          height="250px"
+          src="https://firebasestorage.googleapis.com/v0/b/ieee-amu-7deee.appspot.com/o/IEEE%20Sight%20png.png?alt=media&token=79a11933-883d-432e-9ab6-e3bb59a825e6"
```
Comment (pnijhara) [inline pages/sight.vue:7]: @Arsh-q can we add swiper here also?

File: pages/sight.vue [hunk 2]
```diff
@@ -101,10 +148,106 @@
...
+
+            <v-card-text
+              class="text--primary"
+              style="
+                font-family: 'Hammersmith One', sans-serif !important;
+                font-size: 15px;
+              "
+            >
+              Have an idea which you would love to implement? Email us at
+              amuieee.zhcet@gmail.com Subject of Email : Start-AGroup
```
Comment (pnijhara) [inline pages/sight.vue]: Remove "Start-AGroup"

Comment (pnijhara) [reply pages/sight.vue]: Basically, remove Subject of Email.

File: pages/sight.vue [hunk 3]
```diff
@@ -35,30 +50,62 @@
...
+      <v-card height="600">
+        <v-tabs
+          v-model="tab"
+          height="100"
+          background-color="primary"
+          dark
+          centered
+        >
+          <v-tab
+            v-for="item in items1"
```
Comment (pnijhara) [inline pages/sight.vue:63]: Images are not looking good!

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
@shaista1519 I have requested some changes. Please look at them. Rest all looks good.

File: pages/sight.vue (hunk 3 above)
Comment (shaista1519) [reply pages/sight.vue:63]: These images were provided by the sight coordinators. I think we should keep them for some time.

Comment (shaista1519) [reply pages/sight.vue:63]: Modified the size of the images.

File: pages/sight.vue (hunk 2 above)
Comment (pnijhara) [reply pages/sight.vue]: @shaista1519 please look at this also.

Comment (shaista1519) [reply pages/sight.vue]: done!!

**Review by pnijhara — APPROVED** [review]

**Pull request merged**